lib/ramble/ramble/application.py

```diff
@@ -604,6 +605,31 @@ def add_expand_vars(self, workspace):
 
         self._vars_are_expanded = True
 
+    def add_expand_vars(self, workspace):
+        """Add application specific expansion variables
+
+        Applications require several variables to be defined to function properly.
+        This method defines these variables, including:
+        - command: set to the commands needed to execute the experiment
+        - spack_setup: set to an empty string, so spack applications can override this
+        """
+        if self._vars_are_expanded:
```

Comment:
Is this still used? I don't see it being set in any of the add expand vars functions.